examples/java/src/main/java/org/apache/beam/examples/complete/AutoComplete.java

```diff
@@ -494,7 +494,7 @@ public static void main(String[] args) throws IOException {
                .to(tableRef)
                .withSchema(FormatForBigquery.getSchema())
                .withCreateDisposition(BigQueryIO.Write.CreateDisposition.CREATE_IF_NEEDED)
-               .withWriteDisposition(BigQueryIO.Write.WriteDisposition.WRITE_TRUNCATE));
+               .withWriteDisposition(BigQueryIO.Write.WriteDisposition.WRITE_APPEND));
```

Comment:
I've marked the inputFile option as required, as it is used in both the streaming and batch configurations. This should fix the NPE.